modules/ref_arch_setup/tasks/slv_275.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I assume you didn't mean to include this file

Reply:
I just included it as a basic example similar to the example in the doc. It is not intended to be merged.

Reply:
Removed